### ISSUE TITLE
CSS background colors are ignored

### ISSUE THREAD

**Issue by foundryspatial-duncan** [arachnys/athenapdf#61]
I've got a page with some SVG elements as well as regular HTML. I'm impressed to see the SVG stuff seems to render as expected* but the background colors of ordinary divs are being discarded, even without the `--no-background` commandline argument.

*by "as expected," I mean "same as in the browser."

This is with athenapdf version 2.7.0

**Comment by MrSaints** [comment]
Hey @foundryspatial-duncan, do you have a sample I can look at?

**Comment by foundryspatial-duncan** [comment]
I put together a quick demo here: https://athena-bug-demo.surge.sh/
If I generate a pdf using this command:
`docker run --rm -v $(pwd):/converted/ arachnysdocker/athenapdf athenapdf 'https://athena-bug-demo.surge.sh' athenaTest.pdf`
The PDF renders the same as in the browser, except the right-most div (`div#e`), which is styled from a document-level stylesheet, shows up as white instead of orange.

I actually noticed this behavior when I tired it on a page with no document-level styles. I was expecting more of the divs on the example page to be blank. :P The page where I saw it happening a lot is a non-public app backend, but I'll let you know if I figure out a pattern to recreate it.
